pandora-client-web/src/ui/screens/spacesSearch/spacesSearch.tsx

```diff
@@ -43,6 +43,7 @@ const TIPS: readonly string[] = [
 	`Every single change in the wardrobe happens instantly and is immediately visible to everyone in the room.`,
 	`Public spaces without an admin online inside are not publicly listed in the spaces search.`,
 	`The character context menu can still be opened from a room item's menu while a character is inside.`,
+	`You do not have to first click into the chat input field to start typing in it.`,
```

Comment:
order of the words feels confusing to me somehow

Reply:
I will reorder it tomorrow.

Reply:
Changed.